app/decorators/staff/time_slots_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class TimeSlotsDecorator < Draper::CollectionDecorator
+class Staff::TimeSlotsDecorator < Draper::CollectionDecorator
```

Comment:
That's a good point about consistency overall. I was only looking at the decorators directory itself.